dev/selinux-resolv-conf.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+if [ "$EUID" -ne 0 ]; then
+   echo "This script must be run as root"
+   exit 1
+fi
+
+
+
+##CentOS 7 uses NetworkManager for the network, the ones that dont have static (manually) configured network.
```

Comment:
Is this CentOS or SELinux related?  If the former, change script name and decouple from the the other selinux stuff?

Reply:
done.